packages/odyssey-react-mui/src/DataTable/DataTable.tsx

```diff
@@ -639,6 +639,16 @@ const DataTable = ({
     [columnIds],
   ) as string[];
 
+  const shouldDisplayRowActions = useMemo(
+    () =>
+      (hasRowReordering === true && onReorderRows) ||
+      rowActionButtons ||
+      rowActionMenuItems
+        ? true
+        : false,
+    [hasRowReordering, onReorderRows, rowActionButtons, rowActionMenuItems],
+  );
```

Comment:
Am I misunderstanding this code 😆?

```suggestion
  const shouldDisplayRowActions = useMemo(
    () =>
      (hasRowReordering === true && onReorderRows) ||
      rowActionButtons ||
      rowActionMenuItems,
    [hasRowReordering, onReorderRows, rowActionButtons, rowActionMenuItems],
  );
```

Reply:
Also this:
```suggestion
  const hasVisibleActionRows = useMemo(
    () =>
      (hasRowReordering === true && onReorderRows) ||
      rowActionButtons ||
      rowActionMenuItems
        ? true
        : false,
    [hasRowReordering, onReorderRows, rowActionButtons, rowActionMenuItems],
  );
```